### PR TITLE
Add new ruleset for www.uvnc.com

### PR DESCRIPTION
**Pull request by TotalCaesar659**
https://github.com/EFForg/https-everywhere/issues/15777

### CODE REVIEW

File: src/chrome/content/rules/Uvnc.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	Nonfunctional hosts in *.uvnc.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Uvnc.com">
+	<target host="uvnc.com" />
```
Comment (ivysrono) [inline src/chrome/content/rules/Uvnc.com.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586111710#L319
No ^

Comment (TotalCaesar659) [reply src/chrome/content/rules/Uvnc.com.xml]: What do I need to do? Where to set this "^"? BTW, it's redirecting to https://www.uvnc.com/, maybe this ruleset isn't needed anymore.

Comment (ivysrono) [reply src/chrome/content/rules/Uvnc.com.xml]: ^ means `uvnc.com`.
Travis test tell us there is no `uvnc.com`
Remove `<target host="uvnc.com" />` to comment.

File: src/chrome/content/rules/Uvnc.com.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!--
+	Time out:
+		forum2.uvnc.com
+		mail.uvnc.com
+		support4.uvnc.com
+-->
+<ruleset name="Uvnc.com">
+	<target host="uvnc.com" />
+	<target host="www.uvnc.com" />
+	<target host="forum.uvnc.com" />
```
Comment (ivysrono) [inline src/chrome/content/rules/Uvnc.com.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586547585#L328
Mismatched: forum, www.mail, sponsor4

Comment (TotalCaesar659) [reply src/chrome/content/rules/Uvnc.com.xml]: How to disable them? Just move to comment?

**Pull request merged**